### PR TITLE
Milestone 0.18 release

### PR DESCRIPTION
**Pull request by Misty-W**
## Description

Updating documentation for Milestone 0.18 release
---

### License

- [X] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+## Version 0.18.0
+
+### Summary
+
+This release cycle focused on review and approval of two RFCs, one for Readout Error Mitigation (REM) [#1387](https://github.com/unitaryfund/mitiq/issues/1387) and one for Identity insertion noise scaling [#335](https://github.com/unitaryfund/mitiq/issues/335) (not listed as PRs). It also includes bug fixes and minor dependency upgrades. 
+
+### All Changes
+
+- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
+#1477 by [@dependabot]
```
Comment (andreamari) [inline CHANGELOG.md:12]:
```suggestion
### All Changes

- Update amazon-braket-sdk requirement from ~=1.29.2 to ~=1.29.3 . #1481 by [@dependabot] 

- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
#1477 by [@dependabot] 
```
I merged a new dependabot PR (I should have probably waited), so we need new line here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+### All Changes
+
+- Update black requirement from ~=22.6 to ~=22.8  infrastructureFor issues related to building, packaging, and continuous integration.
+#1477 by [@dependabot] 
+
+- docs: add Aaron-Robertson as a contributor for ideas, code, doc #1464 by [@allcontributors]
+
+- docs: add 1ucian0 as a contributor for infra, code #1461 by [@allcontributors]
+
+- Update amazon-braket-sdk requirement from ~=1.29.0 to ~=1.29.2  infrastructureFor issues related to building, packaging, and continuous integration. #1459 by [@dependabot]
```
Comment (andreamari) [inline CHANGELOG.md:18]: Maybe we can manually remove all the "`infrastructureFor issues related to building, packaging, and continuous integration`" after each dependabot commit.  If we want to keep them, we need a space `infrastructureFor` ---> `infrastructure for `.

**Review by andreamari — APPROVED** [review]
Thanks Misty! Approved with small suggestions.

**Comment by natestemen** [comment]
Just FYI there is a single `git` command in https://mitiq.readthedocs.io/en/stable/release.html#update-the-changelog that can generate the changelog in a slightly automated way (added recently in https://github.com/unitaryfund/mitiq/pull/1404). Once you get the output of the command, you need to change the authors to ensure they are `@`ed, but it makes the process a little easier.

**Pull request merged**